examples/evs-csi-plugin/kubernetes/normal/pvc.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: csi-pvc-evs-shareable.ap-southeast-1b
+spec:
+  accessModes:
+    - ReadWriteMany
+  volumeMode: Block
+  resources:
+    requests:
+      storage: 12Gi
+  storageClassName: evs-share-sc.ap-southeast-1b
```

Comment:
pls add a blank line at the end

Reply:
done, has changed in branch "feat-documents-yaml"